src/test/java/andreeaG/tests/tema7/BaseTest.java

```diff
@@ -0,0 +1,26 @@
+package andreeaG.tests.tema7;
+
+import andreeaG.driver.BrowserManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+
+    protected ChromeDriver driver;
+
+    @BeforeMethod(groups = {"redirect", "login"})
+    public void initChromeBrowser() {
+        driver = BrowserManager.createChromeDriver();
+        driver.get("https://katalon-demo-cura.herokuapp.com/");
+        System.out.println("Executed before method");
+    }
+
+    @AfterMethod(groups = {"redirect", "login", "Home Button"})
```

Comment:
Please set also attribute -> alwaysRun = true on this AfterMethod in order to avoid leaving the browser open no matter what happens. Thanks!